pkg/setting/setting.go

```diff
@@ -957,6 +957,9 @@ func (cfg *Cfg) loadConfiguration(args CommandLineArgs) (*ini.File, error) {
 	// apply command line overrides
 	cfg.applyCommandLineProperties(commandLineProps, parsedFile)
 
+	// migrate install plugins to preinstall
+	cfg.migrateInstallPluginsToPreinstall(parsedFile, os.Getenv("GF_INSTALL_PLUGINS"))
```

Comment:
I'd do this in the function `readPluginSettings`, where all the other preinstall-related logic is. To have everything in the same place.